scipy/optimize/_slsqp_py.py

```diff
@@ -221,25 +220,29 @@ def _minimize_slsqp(func, x0, args=(), jac=None, bounds=None,
                     maxiter=100, ftol=1.0E-6, iprint=1, disp=False,
                     eps=_epsilon, callback=None, finite_diff_rel_step=None,
                     workers=None, **unknown_options):
-    """
+    r"""
     Minimize a scalar function of one or more variables using Sequential
     Least Squares Programming (SLSQP).
 
     Parameters
     ----------
+    maxiter : int, optional
+        Maximum number of iterations. Default value is 100.
     ftol : float
         Precision target for the value of f in the stopping criterion. This value
         controls the final accuracy for checking various optimality conditions;
         gradient of the lagrangian and absolute sum of the constraint violations
         should be lower than ``ftol``. Similarly, computed step size and the
         objective function changes are checked against this value. Default is 1e-6.
-    eps : float
-        Step size used for numerical approximation of the Jacobian.
+    iprint : int
```

Comment:
My preference for altering the docstring would be to avoid describing `iprint`. The rationale for this is that:

- user has fuller control over where the printing goes via a callback. e.g. it can go to a logger, etc.
- the callback approach is more uniform over all the minimize methods, so if you implement a good 'printer' then it can be used widely.

Can you remove this little bit describing `iprint`.

Reply:
done!